internal/mocha_node_test/BUILD

```diff
@@ -0,0 +1,47 @@
+# Copyright 2017 The Bazel Authors. All rights reserved.
```

Comment:
Better to nest tests under `/internal/mocha_node_test/test/...` like `/internal/npm_install/test`. Also, if the test can also act as an example of how to use the rule than it can go under `/examples`